CellProfiler64.iss

```diff
@@ -45,7 +45,6 @@ Name: "{userappdata}\CellProfiler\ijplugins"; Flags: uninsneveruninstall
 Source: ".\dist\CellProfiler.exe"; DestDir: "{app}"; Flags: ignoreversion
 Source: ".\dist\analysis_worker.exe"; DestDir: "{app}"; Flags: ignoreversion
 Source: ".\dist\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs createallsubdirs
-Source: ".\windows\vcredist_x64.exe"; DestDir: "{tmp}"
```

Comment:
I changed the way we do the redistributable according to directions here:
http://www.py2exe.org/index.cgi/Tutorial#Step52

There's a step (https://github.com/CellProfiler/CellProfiler/blob/features/py2exe/setup.py#L236) in the custom py2exe command that goes looking for MSVC 9.0 (or MSVC 9.0 express) in the Windows registry and then finds the redistributable files there. This has fewer chances of conflict with the system copy of the .dll files and it means we don't have to unpack vcredist_x64.exe and run it on the user's machine with some remote possibility that we break some application on that machine.